createSvgIcon refactor in @material-ui 4.9.9 breaking builds

| Tech                 | Version |
| -------------------- | ------- |
| @material-ui/pickers |   v4.0.0-alpha.4   |
| material-ui          |  v4.9.9       |


## Expected behavior

Use createSvgIcon from utils export


`import { createSvgIcon } from '@material-ui/core/utils';`

instead of 

`import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';`

@ https://github.com/mui-org/material-ui-pickers/blob/574944acbb6e3918803914078674d9d815fc260c/lib/src/_shared/icons/ArrowDropDownIcon.tsx#L2


## Actual behavior

```
Module not found: Can't resolve '@material-ui/core/internal/svg-icons/createSvgIcon' in '/.../node_modules/@material-ui/pickers'
```

Refactored utility `createSvgIcon` in latest material-ui version breaks pickers svgs. This has been removed from the internal path in the following commit

https://github.com/mui-org/material-ui/commit/8ea2df8a18fb4a1f8a89e755e0d7f05a9b8eed73

--- a/docs/package.json
+++ b/docs/package.json
@@ -21,7 +21,7 @@
     "@date-io/hijri": "^2.2.0",
     "@date-io/jalaali": "^2.0.0",
     "@mapbox/rehype-prism": "^0.4.0",
-    "@material-ui/core": "^4.9.7",
+    "@material-ui/core": "^4.9.9",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/pickers": "^4.0.0-alpha.1",
     "@types/fuzzy-search": "^2.1.0",

--- a/lib/package.json
+++ b/lib/package.json
@@ -32,7 +32,7 @@
     "email": "dmtr.kovalenko@outlook.com"
   },
   "peerDependencies": {
-    "@material-ui/core": "^4.0.0",
+    "@material-ui/core": "^4.9.9",
     "@types/react": "^16.8.6",
     "react": "^16.8.4",
     "react-dom": "^16.8.4"
@@ -77,7 +77,7 @@
     "@babel/preset-env": "^7.6.0",
     "@babel/preset-react": "^7.0.0",
     "@cypress/webpack-preprocessor": "^4.1.1",
-    "@material-ui/core": "^4.9.7",
+    "@material-ui/core": "^4.9.9",
     "@material-ui/icons": "^4.9.1",
     "@types/enzyme": "^3.10.5",
     "@types/enzyme-adapter-react-16": "^1.0.3",

--- a/lib/rollup.config.js
+++ b/lib/rollup.config.js
@@ -16,6 +16,7 @@ const input = './src/index.ts';
 const globals = {
   react: 'React',
   'react-dom': 'ReactDOM',
+  '@material-ui/core/utils': 'material-ui.utils',
   '@material-ui/core/Button': 'material-ui.Button',
   '@material-ui/core/ButtonBase': 'material-ui.ButtonBase',
   '@material-ui/core/CircularProgress': 'material-ui.CircularProgress',

--- a/lib/src/_shared/icons/ArrowDropDownIcon.tsx
+++ b/lib/src/_shared/icons/ArrowDropDownIcon.tsx
@@ -1,4 +1,4 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import { createSvgIcon } from '@material-ui/core/utils';
 
 export const ArrowDropDownIcon = createSvgIcon(<path d="M7 10l5 5 5-5z" />, 'ArrowDropDownIcon');

--- a/lib/src/_shared/icons/ArrowLeftIcon.tsx
+++ b/lib/src/_shared/icons/ArrowLeftIcon.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import { createSvgIcon } from '@material-ui/core/utils';
 
 export const ArrowLeftIcon = createSvgIcon(
   <>

--- a/lib/src/_shared/icons/ArrowRightIcon.tsx
+++ b/lib/src/_shared/icons/ArrowRightIcon.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import { createSvgIcon } from '@material-ui/core/utils';
 
 export const ArrowRightIcon = createSvgIcon(
   <>

--- a/lib/src/_shared/icons/ClockIcon.tsx
+++ b/lib/src/_shared/icons/ClockIcon.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import { createSvgIcon } from '@material-ui/core/utils';
 
 export default createSvgIcon(
   <React.Fragment>

--- a/lib/src/_shared/icons/DateRangeIcon.tsx
+++ b/lib/src/_shared/icons/DateRangeIcon.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import { createSvgIcon } from '@material-ui/core/utils';
 
 export const DateRangeIcon = createSvgIcon(
   <>

--- a/lib/src/_shared/icons/KeyboardIcon.tsx
+++ b/lib/src/_shared/icons/KeyboardIcon.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import { createSvgIcon } from '@material-ui/core/utils';
 
 export const KeyboardIcon = createSvgIcon(
   <>

--- a/lib/src/_shared/icons/PenIcon.tsx
+++ b/lib/src/_shared/icons/PenIcon.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import { createSvgIcon } from '@material-ui/core/utils';
 
 export const PenIcon = createSvgIcon(
   <path d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z" />,

--- a/lib/src/_shared/icons/TimeIcon.tsx
+++ b/lib/src/_shared/icons/TimeIcon.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import createSvgIcon from '@material-ui/core/internal/svg-icons/createSvgIcon';
+import { createSvgIcon } from '@material-ui/core/utils';
 
 export const TimeIcon = createSvgIcon(
   <>

--- a/lib/typings.d.ts
+++ b/lib/typings.d.ts
@@ -4,11 +4,3 @@ declare module '@date-io/type' {
 
   export type DateType = Moment | DateTime | Date;
 }
-
-declare module '@material-ui/core/internal/svg-icons/createSvgIcon' {
-  import * as React from 'react';
-  import { SvgIconProps } from '@material-ui/core';
-
-  declare const createSvgIcon: (path: React.ReactNode, name: string) => React.FC<SvgIconProps>;
-  export default createSvgIcon;
-}

